src/components/AccountDrawer/HideSpamToggle.tsx

```diff
@@ -0,0 +1,19 @@
+import { t } from '@lingui/macro'
+import { useAtom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+import { SettingsToggle } from './SettingsToggle'
+
+export const hideSpamAtom = atomWithStorage<boolean>('hideSmallBalances', true)
+
+export function HideSpamToggle() {
```

Comment:
add unit test for this component please

Reply:
I think that would be a bit redundant considering this is just an implementation of `SettingsToggle`, but I can add one for `SettingsToggle`